examples/rust_reverse/test/test_rust_reverse.rb

```diff
@@ -12,8 +12,12 @@ def test_it_does_something_useful
   end
 
   def test_stressing_it_out
-    10000.times do
-      assert_equal "a" * 100000, RustReverse.reverse("a" * 100000)
+    GC.stress = true
+
+    expected = "a" * 10000
+
+    1000.times do
+      assert_equal expected, RustReverse.reverse("a" * 10000)
     end
```

Comment:
You are missing `GC.stress = false` to avoid the `GC.stress = true` side-effect affecting all following tests.

Reply:
Other than that, LGTM